src/components/atoms/Avatar/index.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from "react"
+import * as AvatarPrimitive from "@radix-ui/react-avatar"
+
+import { cn } from "@/lib/utils"
+
+const variants = {
+  size: {
+    sm: "h-8 w-8 text-xs",
+    md: "h-10 w-10 text-sm",
+    lg: "h-12 w-12 text-base",
+    xl: "h-14 w-14 text-lg",
+    xxl: "h-16 w-16 text-xl",
+  },
+  variant: {
+    isRounded: {
+      yes: "rounded-full",
+      no: "rounded-none",
+    },
+  },
+  defaultVariants: {
+    size: "md",
+    variant: {
+      color: "default",
+      isRounded: "yes",
+    },
+  },
+}
+
+interface SizeProps {
+  size: 'sm' | 'md' | 'lg' | 'xl' | 'xxl'
+}
+
+interface ColorProps {
+  colorClass?: string
+}
+
+interface RoundedProps {
+  isRounded?: 'yes' | 'no'
+}
```

Comment:
instead of a flag property, how will it be to actually have shapes like rounded or square as props that will allow this component to have extensibility for adding new shapes in future